scenes/game/local_game.gd

```diff
@@ -2468,6 +2574,14 @@ func is_effect_condition_met(performing_player : Player, effect, local_condition
 		elif condition == "life_equals":
 			var amount = effect['condition_amount']
 			return performing_player.life == amount
+		elif condition == "discarded_matches_attack_speed":
+			var discarded_card_ids = effect['discarded_card_ids']
+			assert(discarded_card_ids.size() == 1)
+			var card = card_db.get_card(discarded_card_ids[0])
+			var speed_of_discarded = card.definition['speed']
+			var attack_card = active_strike.get_player_card(performing_player)
+			var printed_speed_of_attack = attack_card.definition['speed']
```

Comment:
this effect only cares about printed speeds right? (i.e. if you have light in play)

Reply:
yes, I double checked the faq